docs/src/configuration/services/elasticsearch.md

```diff
@@ -32,6 +32,18 @@ To switch to OpenSearch, follow the same procedure as for [upgrading](#upgrading
 Elasticsearch 7.9 isn't available in the EU-1 and US-1 regions.
 Consider [region migration](../../guides/general/region-migration.md) if your project is in those regions.
 
+## Use a framework
```

Comment:
Can you describe why you don't have this as the first section? I'm not disagreeing with it, just trying to understand it. I'd worry it might get lost, but hearing the other perspective would be useful.